pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -331,6 +331,18 @@ func testPodDevices(t *testing.T, dataDir, deviceName string, allDevices bool) {
 	assert.Equal(t, 1, len(blkInitCont.VolumeDevices))
 	blkMetaInitCont = deployment.Spec.Template.Spec.InitContainers[8]
 	assert.Equal(t, 1, len(blkMetaInitCont.VolumeDevices))
+
+	// // Test tune Fast settings when OSD on PVC
```

Comment:
nit: no need for double comments
```suggestion
	// Test tune Fast settings when OSD on PVC
```